Block/Bordered/styles.scss

```diff
@@ -0,0 +1,13 @@
+@import '../../css/settings';
+@import '../../css/mixins/index';
+
+.block--bordered {
+  background: map-get($colors, white);
```

Comment:
Let's add `box-sizing: border-box` so the borders don't make it overflow outside the grid